packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -242,19 +242,40 @@ export class SummarizerNode implements IRootSummarizerNode {
         );
         return { latestSummaryUpdated: true, wasSummaryTracked: false, snapshot: snapshotTree };
     }
-
+/**
+ * Called when we get an ack from the server for a summary we've just sent. Updates the reference state of this node
+ * from the state in the pending summary queue.
+ * @param proposalHandle - Handle for the current proposal.
+ * @param referenceSequenceNumber -  reference sequence number of sent summary.
+ */
     protected refreshLatestSummaryFromPending(
         proposalHandle: string,
         referenceSequenceNumber: number,
+        key?: string,
+        parentLatestSummary?: SummaryNode,
     ): void {
         const summaryNode = this.pendingSummaries.get(proposalHandle);
         if (summaryNode === undefined) {
-            // This should only happen if parent skipped recursion AND no prior summary existed.
-            assert(
-                this._latestSummary === undefined,
-                0x1a6 /* "Not found pending summary, but this node has previously completed a summary" */,
-            );
-            return;
+            if (this._latestSummary !== undefined) {
+                this.wipSummaryLogger?.sendTelemetryEvent({
+                    eventName: "SummaryNodeWithoutPendingSummary",
+                    proposalHandle,
+                    referenceSequenceNumber,
+                });
```

Comment:
Still need to check whether this logging works and perhaps add additional info to help understand the scenario.